charts/user-permissions/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
I don't see any of the things defined here being used in the templates... Is there magic somewhere in Helm that hooks into these definitions?